src/common/Axis/LinearAxis/LinearAxisTickLabel.tsx

```diff
@@ -72,14 +75,27 @@ export const LinearAxisTickLabel: FC<Partial<LinearAxisTickLabelProps>> = ({
   }
 
   function getOffset() {
-    const adjustedPadding = typeof padding === 'number' ? { fromAxis: padding, alongAxis: padding } : padding;
+    const adjustedPadding =
+      typeof padding === 'number'
+        ? { fromAxis: padding, alongAxis: padding }
+        : padding;
 
     const spacing = getTickLineSpacing();
-    const offset1 = position === 'start' ? spacing[0] - adjustedPadding.fromAxis : position === 'end' ? spacing[1] + adjustedPadding.fromAxis : 0;
+    const offset1 =
+      position === 'start'
+        ? spacing[0] - adjustedPadding.fromAxis
+        : position === 'end'
+          ? spacing[1] + adjustedPadding.fromAxis
+          : 0;
 
     const align = getAlign();
-    let offset2 = rotation === true ? -5 : 0;
-    offset2 += align === 'center' ? 0 : align === 'start' ? -adjustedPadding.alongAxis : adjustedPadding.alongAxis;
+    let offset2 = 0;
```

Comment:
The only change here is removing this random `-5` value. Tested w/ all rotations and seems better w/o it.